### ISSUE TITLE
HNMs and Endgame monsters.

### ISSUE THREAD

**Issue by Miscbox** [myera/issues#13]
I'll edit this list as I come across things, but a number of HNMs, land, sky, sea and otherwise are once again broken and exploitable due to not being resistant or immune to CC as well as due to movement speed bonus to normal characters vs original movement values of the mob.

These are all on a varied time scale and my own personal experience with them. Not sure what has changed since I did them.

Also I know that a few members of the GM team are aware of this. It's not a complaint but more so a documentation in good faith. If detailed tickets will allow fair play to come along quicker then I will put in the work to let you know what's broken.

Behemoth/KingBehemoth: Sleepable, bindable. Gravity with no resistance building. Easily Kited without gravity anyway to the point where a ranger can solo dps with Anni delay by walking out of range and getting shots off.

Fafnir/Nidhogg: Too easily sleepable.

Jailer of Faith: Easily Silenceable, Sleepable.

Jailer of Fortitude: Easily Sleepable.

Jailer of Love: Easily Sleepable, bindable, gravable.

Jailer of Prudence: Sleepable.

Ix'drk: Sleepable (last time I did it)

Ix'drg: Sleepable,  Bindable.

Hydra/Cerb: same deal.

Sky gods: Sleepable

Long story short - A revision of most of the NMs in the game at this point, namely the ones with the rarer drops / and trigger mobs is necessary.

You can contact me in game on the character Izayoi and I would be glad to spend the time with you travelling the world to test monsters that come to mind.

**Comment by Scavenge** [comment]
definitely a problem.  but shuold be easily fixed by our custom files in which I originally added at least 75% of these to be resistant or immune.

**Comment by Leelink1** [comment]
Lets add Jormy to the list as well.

**Comment by Miscbox** [comment]
What do you mean by adding Jorm? I guess it would make sense to give it enhanced movement speed since it doesn't have draw in but it's meant to be sleepable and some other things. Plus he's not really that exploitable since regen and def/mdef would make it a terrible fight to bind nuke/low man without constant dps...esp on this server lol.

**Comment by Leelink1** [comment]
Apprently he was able to just be Sleep nuked with no issues.

**Comment by Miscbox** [comment]
Interesting...maybe raise its resist rate and regen a bit?

**Comment by GM-Dave** [comment]
Sea NMs and Jailers (with the exception of Absolute Virtue) are working as intended mostly with correct drop tables.

Faf & Nid are immune to sleep, Behe/KB and Adam/Aspid are either immune or build resistance to sleep.

Wyrms all are immune to sleep currently as well.

Sky Gods & Kirin are sleepable but will build resistance.

AU HNMs should all currently be immune to sleep & bind

If any issues are still to be found please file a new ticket.
